app/views/OnboardingView/index.js

```diff
@@ -112,87 +65,38 @@ class OnboardingView extends React.Component {
 		}
 	}
 
-	renderClose = () => {
-		const { theme } = this.props;
-		if (this.previousServer) {
-			let top = 15;
-			if (isIOS) {
-				top = isNotch ? 45 : 30;
-			}
-			return (
-				<TouchableOpacity
-					style={[styles.closeModal, { top }]}
-					onPress={this.close}
-					testID='onboarding-close'
-				>
-					<CustomIcon
-						name='cross'
-						size={30}
-						color={themes[theme].actionTintColor}
-					/>
-				</TouchableOpacity>
-			);
-		}
-		return null;
-	}
-
 	render() {
 		const { theme } = this.props;
 		return (
-			<SafeAreaView
-				style={[
-					styles.container,
-					{ backgroundColor: themes[theme].backgroundColor }
-				]}
-				testID='onboarding-view'
-			>
-				<StatusBar theme={theme} />
-				<Image style={styles.onboarding} source={{ uri: 'onboarding' }} fadeDuration={0} />
-				<Text style={[styles.title, { color: themes[theme].titleText }]}>{I18n.t('Welcome_to_RocketChat')}</Text>
-				<Text style={[styles.subtitle, { color: themes[theme].auxiliaryText }]}>{I18n.t('Open_Source_Communication')}</Text>
-				<View style={[styles.buttonsContainer, isTablet && sharedStyles.tabletScreenContent]}>
-					<Button
-						type='secondary'
-						title={I18n.t('Connect_to_a_server')}
-						icon={<CustomIcon name='permalink' size={30} color={themes[theme].actionTintColor} />}
-						onPress={this.connectServer}
-						testID='connect-server-button'
-						theme={theme}
-					/>
-					<Button
-						type='secondary'
-						title={I18n.t('Join_the_community')}
-						subtitle='open.rocket.chat'
-						icon={<Image source={{ uri: 'logo_onboarding' }} style={{ width: 32, height: 27 }} fadeDuration={0} />}
-						onPress={this.joinCommunity}
-						testID='join-community-button'
-						theme={theme}
-					/>
-					<Button
-						type='primary'
-						title={I18n.t('Create_a_new_workspace')}
-						icon={<CustomIcon name='plus' size={30} color={themes[theme].buttonText} />}
-						onPress={this.createWorkspace}
-						testID='create-workspace-button'
-						theme={theme}
-					/>
-				</View>
-				{this.renderClose()}
-			</SafeAreaView>
+			<FormContainer theme={theme}>
+				<FormContainerInner>
+					<Image style={styles.onboarding} source={{ uri: 'logo' }} fadeDuration={0} />
```

Comment:
Check the difference between these two images:
### OnboardView
![Screen Shot 2020-03-30 at 09 20 28](https://user-images.githubusercontent.com/29778115/77911747-b7daf700-7267-11ea-909f-60f4dacc1282.png)
### WorkspaceView
![Screen Shot 2020-03-30 at 09 20 32](https://user-images.githubusercontent.com/29778115/77911750-ba3d5100-7267-11ea-93e3-4ecb6c3aeea2.png)